<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>427</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>427</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>427</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VoiceSockets</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/logo.png</normaloff>assets/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mute_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mute</string>
    </property>
   </widget>
   <widget class="QListWidget" name="users">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="channel_selct" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select channel</string>
     </property>
    </widget>
    <widget class="QComboBox" name="channel_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="ico">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/vc.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="start_settings_select" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="nickname_input">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nickname</string>
     </property>
    </widget>
    <widget class="QComboBox" name="micro">
     <property name="geometry">
      <rect>
       <x>78</x>
       <y>70</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Micro</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="on_voice_chat" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QLabel" name="statuslabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Statuslabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mute_selected">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>291</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>mute selected user</string>
     </property>
    </widget>
    <widget class="QComboBox" name="channel_selecter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>291</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
